src/lib/pages/backoffice/Loan/LoanDetails/LoanOverbookedWarning/LoanOverbookedWarning.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withCancel } from '@api/utils';
+import { documentApi } from '@api/documents';
+
+export default class LoanOverbookedWarning extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      overbooked: false,
+    };
+  }
+
+  componentDidMount() {
+    this.fetchCirculation()
+  }
+
+  componentWillUnmount() {
+    this.cancellableCirculation && this.cancellableCirculation.cancel();
+  }
+
+  fetchCirculation = async () => {
+    const { documentPid } = this.props;
+    try {
+      this.cancellableCirculation = withCancel(
+        documentApi.get(documentPid)
+      );
+      const response = await this.cancellableCirculation.promise;
+      const circulation = response.data.metadata.circulation;
+      this.setState({ overbooked: circulation.overbooked});
+    } catch (error) {
+      if (error !== 'UNMOUNTED') {
+        this.setState({ overbooked: false});
+      }
+    }
+  };
+
+
+  render() {
+    const { overbooked } = this.state;
+    const { documentPid } = this.props;
+    return (
+      <>
+      {overbooked &&
+        <div class="ui warning message">
```

Comment:
why didn't you use the Semantic UI Message component instead of the custom div ? we also have our own wrapper component to unify the look in the application - please check the available components in the code or semantic UI documentation